src/bootstrap-table.js

```diff
@@ -2335,10 +2340,16 @@
     };
 
     BootstrapTable.prototype.updateRow = function (params) {
-        if (!params.hasOwnProperty('index') || !params.hasOwnProperty('row')) {
-            return;
-        }
-        $.extend(this.data[params.index], params.row);
+        var me = this;
```

Comment:
Use `var that = this;` in order to follow the standard. **Don't forget change the variable name in the function**